src/main/java/de/vitagroup/num/service/UserService.java

```diff
@@ -288,18 +295,57 @@ private void addRoles(User user) {
     return users;
   }
 
-  private List<Notification> collectNotification(String userId) {
+  private List<Notification> collectProfileUpdateNotification(
+      String userId, String loggedInUserId) {
```

Comment:
Isn't that only about user name changes? If yes shouldn't this be renamed to collectUsernameUpdate....?